Sources/XMTPiOS/Group.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Group.swift
+//
+//
+//  Created by Pat Nakajima on 2/1/24.
+//
+
+import Foundation
+import LibXMTP
+
+public struct Group: Identifiable, Equatable, Hashable {
+	var ffiGroup: FfiGroup
+	var client: Client
+
+	struct Member {
+		var ffiGroupMember: FfiGroupMember
+
+		public var accountAddress: String {
+			ffiGroupMember.accountAddress
+		}
+	}
+
+	public var id: Data {
+		ffiGroup.id()
+	}
+
+	public func sync() async throws {
+		try await ffiGroup.sync()
+	}
+
+	public static func == (lhs: Group, rhs: Group) -> Bool {
+		lhs.id == rhs.id
+	}
+
+	public func hash(into hasher: inout Hasher) {
+		id.hash(into: &hasher)
+	}
+
+	public var members: [String] {
```

Comment:
Still think we should name this 
```suggestion
	public var memberAddresses: [String] {
```